pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -338,54 +341,15 @@ func (p *AgentPoller) Stop() {
 	}
 }
 
-// With libvirt 5.6.0 direct call to agent can be replaced with call to libvirt Domain.GetGuestInfo
```

Comment:
Can you modify this comment, to mention that the rest of the commands will be removed in a future PR?

Reply:
Yes. Mentioning that will tell the reader that we don't want to keep this function.